### DISCUSSION TITLE
Use Gunicorn or alternative instead of Flask for ML

### DISCUSSION THREAD

**Discussion by bertmelis** [immich-app/immich#1878] [General]
when starting the new machine learning container, the logs show this warning:

```
WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
```

It's just a warning so it runs fine, but the better option would be to use one of the production servers, like Gunicorn (https://flask.palletsprojects.com/en/2.2.x/deploying/gunicorn/)

**Comment by ollywelch** [comment]
I've started working on this one! Hopefully should be a simple enough fix :)
